### PR TITLE
Pick up rendering stuff and dragon stuff

### PR DESCRIPTION
**Pull request by liach**
Picks from #1026, which appears unmaintained.

Have rebased and tweaked some poor mappings from that pr.

### CODE REVIEW

**Review by Runemoro — APPROVED** [review]

File: mappings/net/minecraft/client/render/RenderLayer.mapping [hunk 1]
```diff
@@ -97,6 +97,13 @@ CLASS net/minecraft/class_1921 net/minecraft/client/render/RenderLayer
 		ARG 1 affectsOutline
 	METHOD method_24295 isOutline ()Z
 	METHOD method_24296 asOptional ()Ljava/util/Optional;
+	METHOD method_24468 getOUtline (Lnet/minecraft/class_2960;Lnet/minecraft/class_4668$class_4671;)Lnet/minecraft/class_1921;
```
Comment (YanisBft) [inline mappings/net/minecraft/client/render/RenderLayer.mapping]: `getOutline`

**Review by YanisBft — APPROVED** [review]

File: mappings/net/minecraft/client/render/RenderLayer.mapping (hunk 1 above)
Comment (liach) [reply mappings/net/minecraft/client/render/RenderLayer.mapping]:
```suggestion
	METHOD method_24468 getOutline (Lnet/minecraft/class_2960;Lnet/minecraft/class_4668$class_4671;)Lnet/minecraft/class_1921;
```

File: mappings/net/minecraft/entity/player/PlayerEntity.mapping [hunk 2]
```diff
@@ -232,6 +232,7 @@ CLASS net/minecraft/class_1657 net/minecraft/entity/player/PlayerEntity
 	METHOD method_7356 getShoulderEntityLeft ()Lnet/minecraft/class_2487;
 	METHOD method_7357 getItemCooldownManager ()Lnet/minecraft/class_1796;
 	METHOD method_7358 wakeUp (ZZ)V
+		ARG 2 shouldUpdate
```
Comment (Juuxel) [inline mappings/net/minecraft/entity/player/PlayerEntity.mapping:235]: Should update what exactly?

File: mappings/net/minecraft/client/render/entity/EnderCrystalEntityRenderer.mapping [hunk 3]
```diff
@@ -1,3 +1,10 @@
 CLASS net/minecraft/class_892 net/minecraft/client/render/entity/EnderCrystalEntityRenderer
+	FIELD field_21002 SINE_FOURTY_FIVE_DEGREES F
```
Comment (Juuxel) [inline mappings/net/minecraft/client/render/entity/EnderCrystalEntityRenderer.mapping:2]:
```suggestion
	FIELD field_21002 SINE_45_DEGREES F
```

File: mappings/net/minecraft/client/render/entity/EnderDragonEntityRenderer.mapping [hunk 4]
```diff
@@ -1,11 +1,41 @@
 CLASS net/minecraft/class_895 net/minecraft/client/render/entity/EnderDragonEntityRenderer
 	FIELD field_21006 EYES_TEX Lnet/minecraft/class_2960;
+	FIELD field_21007 SINE_SIXTY_DEGREES F
```
Comment (Juuxel) [inline mappings/net/minecraft/client/render/entity/EnderDragonEntityRenderer.mapping:3]:
```suggestion
	FIELD field_21007 SINE_60_DEGREES F
```

File: mappings/net/minecraft/client/render/entity/feature/StuckObjectsFeatureRenderer.mapping [hunk 5]
```diff
@@ -4,6 +4,11 @@ CLASS net/minecraft/class_4507 net/minecraft/client/render/entity/feature/StuckO
 	METHOD method_22130 renderObject (Lnet/minecraft/class_4587;Lnet/minecraft/class_4597;ILnet/minecraft/class_1297;FFFF)V
 		ARG 1 matrix
 		ARG 2 vertexConsumers
-		ARG 5 tickDelta
+		ARG 3 light
+		ARG 4 entity
+		ARG 5 dirX
+		ARG 6 dirY
+		ARG 7 dirZ
```
Comment (Juuxel) [inline mappings/net/minecraft/client/render/entity/feature/StuckObjectsFeatureRenderer.mapping:11]: Dir?

**Review by Juuxel — CHANGES_REQUESTED** [review]

**Review by modmuss50 — APPROVED** [review]

File: mappings/net/minecraft/client/render/entity/feature/StuckObjectsFeatureRenderer.mapping (hunk 5 above)
Comment (liach) [reply mappings/net/minecraft/client/render/entity/feature/StuckObjectsFeatureRenderer.mapping:11]: direction i supposed, from the old pr not changed

File: mappings/net/minecraft/client/render/entity/EnderDragonEntityRenderer.mapping (hunk 4 above)
Comment (liach) [reply mappings/net/minecraft/client/render/entity/EnderDragonEntityRenderer.mapping:3]: should have used `HALF_SQRT_3` I guess

**Pull request merged**